tests/functional/page-objects/login-register-page.js

```diff
@@ -1,6 +1,31 @@
 import { Selector } from 'testcafe'
 
+const pageClass = Selector('.RegisterLogin')
+const signUpEmailInput = Selector('.RegisterLogin_signUp input[name="email"]')
+const signUpPasswordInput = Selector('.RegisterLogin_signUp input[name="password"]')
+const signUpConfirmPasswordInput = Selector('.RegisterLogin_signUp input[name="confirmPassword"]')
+const signUpDisclaimer = Selector('.RegisterLogin_signUp input[name="testnet"]')
+const signUpButton = Selector('.RegisterLogin_signUp .submit-button button')
+
 export const LoginRegisterPage = {
```

Comment:
Not sure if improves readability at all, just throwing it out there as an idea.

```js
export const LoginRegisterPage = {
  selectors: {
    pageClass,
    signUpEmailInput,
    signUpPasswordInput,
    signUpConfirmPasswordInput,
    signUpDisclaimer,
    signUpButton,
  },
  actions: {
    signUp: async (t, email, password, options = { skipDisclaimer: false }) => {
      await t
        .typeText(signUpEmailInput, email)
        .typeText(signUpPasswordInput, password)
        .typeText(signUpConfirmPasswordInput, password)

      if (!options.skipDisclaimer) {
        await t.click(signUpDisclaimer)
      }

      await t.click(signUpButton)
    },
  }
}
```

Reply:
I like that approach @kennylavender, let's keep it in mind for the future.

Reply:
I don't find the extra layer of indirection more readable - it's just a POJO.
We don't create object this way, right?
```javascript
const obj = {
  properties: {
    foo: 1,
    bar: 2
  },
  methods: {
    hello: () => 'Hello!'
  }
}